Corona-Warn-App/src/main/res/layout/home_create_event_card.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/spacing_normal"
+        tools:showIn="@layout/home_card_container_layout">
+
+        <TextView
+            android:id="@+id/create_event_card_headline"
+            style="@style/headline5"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:accessibilityHeading="true"
+            android:text="@string/create_event_card_headline"
+            android:lines="2"
+            android:maxLines="2"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+         <TextView
+             android:id="@+id/create_event_card_subtitle"
+             style="@style/subtitleMedium"
+             android:layout_width="0dp"
+             android:layout_height="wrap_content"
+             android:layout_marginTop="@dimen/spacing_small"
+             android:layout_marginEnd="@dimen/spacing_tiny"
```

Comment:
I think the design has bigger spacing between the image and the text

Reply:
Ok, I'll have another look.

Reply:
The design has a spacing of 12 between text and image which we currently don't have in our dimens. A step up from spacing_tiny is spacing_small (8dp to 16dp). I'll update it to that but it'll be a bit too much now. Should we add a 12dp dimen? I've seen many places where the designers use 12dp.